app/views/occupation_standards/show.html.erb

```diff
@@ -114,21 +128,21 @@
                             </div>
                           <% end %>
                         <% else %>
-                          <p class="text-white">No documents available at the moment</p>
+                          <p class="text-white">No documents available at this time</p>
                         <% end %>
-                        <%= link_to occupation_standard_path(@occupation_standard, format: :docx), class: "hidden" do %>
```

Comment:
@FerPerales since I merged my PR first, seems like there's a conflict here, as we don't actually want to revert this change. Will you rebase and undo these removals? Thanks!